alibi_detect/saving/schemas.py

```diff
@@ -49,7 +34,7 @@ def coerce_int2list(value: int) -> List[int]:
         return value
 
 
-class SupportedModelsType:
+class SupportedModels:
```

Comment:
Nitpick: should this (and the optimizer one) be singular as it defines a `SupportedModel` type?